### PR TITLE
stable: rollout 31.20200108.3.0

### PR DESCRIPTION
**Pull request by jlebon**
This is the first release so there's no need to have a long rollout.
Though for fun I did `"duration_minutes": 5` as a test for Cincinnati.

### CODE REVIEW

File: updates/stable.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "stream": "testing",
+  "metadata": {
+    "last-modified": "2020-01-10T17:45:18Z"
+  },
+  "releases": [
+    {
+      "version": "31.20200108.3.0",
+      "metadata": {
+        "rollout": {
+          "start_epoch": 1576765800,
```
Comment (dustymabe) [inline updates/stable.json]:
```
$ date -d @1576765800
Thu 19 Dec 2019 09:30:00 AM EST
```

A date in the past?

Comment (jlebon) [reply updates/stable.json]: Fixed!

**Comment by jlebon** [comment]
Dropped hold on this now!

File: updates/stable.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "stream": "stable",
+  "metadata": {
+    "last-modified": "2020-01-10T17:45:18Z"
+  },
+  "releases": [
+    {
+      "version": "31.20200108.3.0",
+      "metadata": {
+        "rollout": {
+          "start_epoch": 1578690000,
```
Comment (dustymabe) [inline updates/stable.json]:
```
$ date -d @1578690000
Fri 10 Jan 2020 04:00:00 PM EST
```

**Pull request merged**